index.html

```diff
@@ -4,6 +4,22 @@
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>GEO-Bench-2: A Capability-Aware Benchmark for Geospatial Foundation Models</title>
+  <meta name="description" content="A large-scale, capability-aware benchmark for evaluating Geospatial Foundation Models across 19 diverse datasets, measuring progress in perception, reasoning, and generalization.">
+  <meta name="keywords" content="GeoBench 2, Geospatial AI, Foundation Models, Earth Observation, Remote Sensing, AI Alliance, NASA, IBM, TUM, Benchmarking">
+  <meta name="author" content="The AI Alliance">
+
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="https://the-ai-alliance.github.io/GEO-Bench-2/">
+  <meta property="og:title" content="GEO-Bench-2: A New Standard for Geospatial AI">
+  <meta property="og:description" content="We evaluated 14 Foundation Models across 19 datasets. See which models truly understand our planet in the official GEO-Bench-2 release.">
```

Comment:
Perhaps just a personal opinion, but I don't think any of the models "understand" anything, let alone understanding of the entire planet from some classification and segmentation datasets. Maybe somehthing like "See how state-of-the-art models perform across various capabilities in the official GEO-Bench-2 release."